GMPWrapper/python/GMPWrapper/parseConstants.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+def parseConstants(constants):
+  again = True
+  while again:
+    again = False
+    for key,value in constants.items():
+      if type(value) is list:
+        for idx, val in enumerate(value):
+          value[idx] = val % constants
+          again = (again) or (value[idx] != val)
+        formatted_list = ""
+        for val in value:
+          formatted_list += '{} '.format(val)
+        constants[key] = formatted_list
```

Comment:
These four lines are the same as
```suggestion
        constants[key] = " ".join(value)
```
right? (except for a trailing space)